src/test_suite/fuzz_interface.py

```diff
@@ -19,12 +19,19 @@
     - input: The fuzz target Context
     - output: The fuzz target Effects
 - diff_effect_fn: A function that compares two effects messages for equality
+- prune_effects_fn: A function that prunes effects to remove extra fields (e.g. remove accounts that weren't actually modified)
 - human encode/decode functions for the context and effects messages to
   convert the messages to/from human-readable format (in-place).
   Both context and effects messages can have their own encode/decode functions.
 """
 
 
+def generic_effects_prune(
+    ctx: ContextType, effects: dict[str, EffectsType]
```

Comment:
nit: we should probably define and document some type alias for this since it's used everywhere

Reply:
referring to `dict[str, EffectsType]`